builtins/amp-ad.js

```diff
@@ -283,6 +283,10 @@ export function installAd(win) {
      * @private
      */
     noContentHandler_() {
+      if (!this.placeholder_) {
+        this.element.style.height = 0;
```

Comment:
Made it `display:none`

also the `noContentHandler_` method is called with defer mutate so i dont think ill have to do it here again.

`this.deferMutate(this.noContentHandler_.bind(this));`

@dvoytenko any tips on how to adjust scroll position?